accord-core/src/main/java/accord/impl/progresslog/WaitingState.java

```diff
@@ -467,6 +480,8 @@ private void runInternal(SafeCommandStore safeStore, SafeCommand safeCommand, De
         awaitRoute = awaitRoute.slice(roundStart, roundEnd);
         // TODO (desired): use some mechanism (e.g. random chance or another counter)
         //   to either periodically fetch the whole remaining route or gradually increase the slice length
+        if (tracing != null)
+            tracing.trace(owner.commandStore, String.format("Blocked until %s. Waiting for %s to satisfy; round %d of %d.", blockedUntil, awaitRoute, roundIndex, (awaitRoute.size() + (roundSize - 1))/roundSize));
```

Comment:
just to make sure: roundSize can never be 0 right? Checked the code and doesn't seem like it can, but wanted to ask to make sure.